ground/gcs/src/app/main.cpp

```diff
@@ -140,6 +140,34 @@ static inline QString msgCoreLoadFailure(const QString &why)
     return QCoreApplication::translate("Application", "Failed to load core: %1").arg(why);
 }
 
+static inline QStringList getPluginPaths()
```

Comment:
This is code that was removed in a prior PR for simplified packaging that has instead proven necessary.